### PR TITLE
agent flags for blue cert

### PR DESCRIPTION
**Pull request by garunagiri**
Breaking the blue cert PR into a few separate PRs. This one is just the agent flags.

### CODE REVIEW

File: app/flags/agents/blue_0.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name = 'Red agent - *nix'
+challenge = 'Deploy a red agent on a Linux or Darwin machine. This agent will be used in later challenges.'
```
Comment (scottctaylor12) [inline app/flags/agents/blue_0.py:2]: Can you change Darwin to Mac in the challenge description? Users will most likely not understand that darwin == mac.
Or you could put `Mac/Darwin machine`.

File: app/flags/agents/blue_1.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name = 'Blue agent - *nix'
+challenge = 'Deploy a blue agent on a Linux or Darwin machine. Ensure that this agent has ELEVATED priviliges. ' \
```
Comment (scottctaylor12) [inline app/flags/agents/blue_1.py:2]: Same as above, Mac instead of Darwin

File: app/flags/agents/blue_0.py (hunk 1 above)
Comment (garunagiri) [reply app/flags/agents/blue_0.py]: Changed to Mac/Darwin

File: app/flags/agents/blue_1.py (hunk 2 above)
Comment (garunagiri) [reply app/flags/agents/blue_1.py]: Changed to Mac/Darwin

**Pull request merged**